### DISCUSSION TITLE
Zigbee sniff documentation

### DISCUSSION THREAD

**Discussion by chris-1243** [Koenkk/zigbee2mqtt#22998] [General]
I just started to use a Dongle-E to sniff traffic. I strictly follow the documentation which is quite clear but ran into an issue. I would suggest to add a sentence depending on the `ezsp` adapter used.

According to the documentation [Sniffing traffic](https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#_3-sniffing-traffic-1), we should run this command `java -jar ZigbeeSniffer.jar -baud 115200 -flow hardware -port {PORT} -c {CHANNEL}`. For the Dongle-E, as it doesn't have flow control, it looks this `java -jar ZigbeeSniffer.jar -baud 115200 -flow none -port {PORT} -c {CHANNEL}`. I think, it would be great to create such difference for people not aware of this. I spent half an hour to understand what I was doing wrong. Otherwise, it was an easy guide to follow.

I did a pull request following the link provided at the end of the documentation.....

**Comment by Koenkk** [comment]
> I did a pull request following the link provided at the end of the documentation.....

Could you provide a link to it? I couldn't find your pr

**Reply by chris-1243** [reply]
Well as it is the first time I'm trying to help, I guess I messed up something.

I have been asked to get a fork of the repository which I did and I can see it. No idea where my change is. I need to dig a bit more.

If you have some tips to get me through the process, I take them😉

**Comment by chris-1243** [comment]
[2802](https://github.com/Koenkk/zigbee2mqtt.io/pull/2802)

I hope I did it correctly
